apps/blazor-test-app/.gitignore

```diff
@@ -0,0 +1,14 @@
+# TeamsFx files
+node_modules
+.fx/configs/localSettings.json
+.fx/states/*.userdata
+.DS_Store
+.env.teamsfx.local
+subscriptionInfo.json
+build
+.fx/configs/config.local.json
+.fx/states/state.local.json
+appsettings.Development.json
+FakesAssemblies/Microsoft.AspNetCore.Components.Server.6.0.0.0.Fakes.xml
+FakesAssemblies/Microsoft.AspNetCore.Components.Server.6.0.0.0.Fakes.messages
+wwwroot/js/MicrosoftTeams.min.js
```

Comment:
I know that I am a broken record about such things but alphabetizing this list would make it easier for people to look through it to see if something is present.

Reply:
(unless this is an auto generated file)

Reply:
Yes, this was auto generated. Yes, I also fixed it for you.

Reply:
![ezgif com-gif-maker](https://user-images.githubusercontent.com/36546967/219511648-1c240970-9762-45cf-8137-df303041e3dc.gif)